superbench/benchmarks/micro_benchmarks/nvbandwidth.py

```diff
@@ -38,13 +38,11 @@ def add_parser_arguments(self):
 
         self._parser.add_argument(
             '--test_cases',
+            nargs='+',
             type=str,
-            default='',
+            default=[],
             required=False,
-            help=(
-                'Specify the test case(s) to run, either by name or index. By default, all test cases are executed. '
-                'Example: --test_cases 0,1,2,19,20'
-            ),
+            help='Specify the test case(s) to run, either by name or index. By default, all test cases are executed.',
```

Comment:
Do we need to describe a little more about the map between `name` and `index`? Users may don't know how to fill this field.